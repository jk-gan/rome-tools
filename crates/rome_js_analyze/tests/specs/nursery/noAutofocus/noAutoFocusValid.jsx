<>
	<input />
    <input autoFocus={undefined} />
    <Input autoFocus={"false"} />
</>