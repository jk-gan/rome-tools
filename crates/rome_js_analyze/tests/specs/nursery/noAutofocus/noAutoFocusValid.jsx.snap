---
source: crates/rome_js_analyze/tests/spec_tests.rs
expression: noAutoFocusValid.jsx
---
# Input
```js
<>
	<input />
    <input autoFocus={undefined} />
    <Input autoFocus={"false"} />
</>
```


